c4cs/c4cs.github.io
# Travis build schedule

# What it does?
Allows for daily builds of the website using cron jobs

# What to check?
Shell script should never infinite loop, GH tokens are secure, Travis yaml is valid

## How to test?
- Run a regular manually triggered build and ensure it is successful. No subsequent pushes by Travis CI should be observed and no additional builds should be triggered.
- Run a cron job triggered build. Ensure that Travis CI pushes a working commit after the cron job runs.
